### PR TITLE
add section about backlog

### PR DESCRIPTION
**Pull request by davejrt**
Not sure if there's anything missing for the moment here. We don't have a dedicated board/project to manage the backlog at this stage. We can used this section as a lifting off point to fill out later if that's preferable.

### CODE REVIEW

File: handbook/engineering/distribution/index.md [hunk 1]
```diff
@@ -132,3 +132,13 @@ Examples:
 - Measure and sustain 99.99% uptime and reliability of Sourcegraph Cloud.
 - Enforce access controls and network security policies for our Cloud resources.
 - Make it easy for customers to share relevant data when submitting bug reports.
+
+
+### Other processes
+#### Backlog grooming
+
+The distribution team runs a backloog grooming session every two weeks to ensure that backlog is correctly priorotized for the upcoming sprint.
```
Comment (pecigonzalo) [inline handbook/engineering/distribution/index.md:140]: bäckög

Comment (davejrt) [reply handbook/engineering/distribution/index.md]: I wrote it 5 times, I was bound to screw it up at least once.

**Review by pecigonzalo — APPROVED** [review]
I would move it and inline it here @davejrt https://about.sourcegraph.com/handbook/engineering/distribution/recurring_processes.
That might allow us to change the text a bit to make it clear that it h appens in the weeks between planning and so forth.

**Comment by davejrt** [comment]
> I would move it and inline it here @davejrt https://about.sourcegraph.com/handbook/engineering/distribution/recurring_processes.
> That might allow us to change the text a bit to make it clear that it h appens in the weeks between planning and so forth.

Good call...I missed this section altogether on the first pass

File: handbook/engineering/distribution/recurring_processes.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ At the beginning of each cycle, [we determine what work we plan to do](#planning
 1. Revisit the goals. Are they still the right goals?
 1. If there are any remaining uncertainties following the meeting, the owners of each theme or task should follow up before the start of the cycle.
 
+**Backlog grooming:** The distribution team runs a backlog grooming session every two weeks to ensure that backlog is correctly priorotized for the upcoming sprint. We use the backlogs for tracking bugs, small features, and unplanned work. We don’t use the backlog for tracking work that is expressly planned in our roadmap. To add an issue, tag it team/distribution to notify the distribution team.
```
Comment (bobheadxi) [inline handbook/engineering/distribution/recurring_processes.md:43]:
```suggestion
**Backlog grooming:** The distribution team runs a backlog grooming session every two weeks to ensure that backlog is correctly prioritized for the upcoming sprint. We use the backlogs for tracking bugs, small features, and unplanned work. We don’t use the backlog for tracking work that is expressly planned in our roadmap. To add an issue, tag it `team/distribution` to notify the distribution team.
```

**Review by bobheadxi — APPROVED** [review]

File: handbook/engineering/distribution/recurring_processes.md (hunk 2 above)
Comment (bobheadxi) [reply handbook/engineering/distribution/recurring_processes.md:43]: Do we have tooling to notify on team tags? I remember seeing some other teams had it

Comment (bobheadxi) [reply handbook/engineering/distribution/recurring_processes.md:43]: https://github.com/sourcegraph/sourcegraph/issues/17476#issuecomment-763812581

**Pull request merged**